Automattic/wp-calypso
# Move postcss config to calypso-build package

#### Changes proposed in this Pull Request

* Move the postcss.config.js file. 

#### Testing instructions
* Does `npm run build-css` work as expected? No errors.
* Build Calypso. 
* Do the css files contain the new autoprefixed valued added by postcss.

Fixes https://github.com/Automattic/wp-calypso/issues/31362
